packages/integration-test/test/scheduled/erc20-fee-proxy.test.ts

```diff
@@ -79,7 +85,7 @@ describe('ERC20 Fee Proxy detection test-suite', () => {
     expect(params?.to).toBe('0x4E64C2d06d19D13061e62E291b2C4e9fe5679b93');
     expect(balance.events[0].amount).toBe('1000000000000000000');
     expect(balance.events[0].timestamp).toBe(1599070058);
-  });
+  }, 10000);
 
```

Comment:
I'm not sure about increasing this timeout but when I ran the test individually it passes, when I ran all tests at once I get timeout error. What do you guys think ?